mne/viz/brain/view.py

```diff
@@ -0,0 +1,280 @@
+# Authors: Alexandre Gramfort <alexandre.gramfort@telecom-paristech.fr>
+#          Eric Larson <larson.eric.d@gmail.com>
+#          Oleh Kozynets <ok7mailbox@gmail.com>
+#          Guillaume Favelier <guillaume.favelier@gmail.com>
+#
+# License: Simplified BSD
+
+from collections import namedtuple
+import sys
+
+import numpy as np
+
+
+View = namedtuple('View', 'elev azim')
+
+views_dict = {'lateral': View(elev=5, azim=0),
+              'medial': View(elev=5, azim=180),
+              'rostral': View(elev=5, azim=90),
+              'caudal': View(elev=5, azim=-90),
+              'dorsal': View(elev=90, azim=0),
+              'ventral': View(elev=-90, azim=0),
+              'frontal': View(elev=5, azim=110),
+              'parietal': View(elev=5, azim=-110),
+              'lat': View(elev=5, azim=0),
+              'med': View(elev=5, azim=180),
+              'fos': View(elev=5, azim=90),
+              'cau': View(elev=5, azim=-90),
+              'dor': View(elev=90, azim=0),
+              'ven': View(elev=-90, azim=0),
+              'fro': View(elev=5, azim=110),
+              'par': View(elev=5, azim=-110)}
+
+
+class TimeViewer(object):
+    u"""Creates time viewer widget.
+
+    Parameters
+    ----------
+    brain : Brain
+        Object with cortex mesh to be controlled by time
+        viewer.
+    """
+
+    def __init__(self, brain):
+        import ipyvolume as ipv
+        import ipywidgets as widgets
+
+        if brain.data['time'] is None:
+            raise ValueError('Brain class instance does not have time data.')
+
+        self._brain = brain
+        time = brain.data['time']
+        time_idx = brain.data['time_idx']
+        overlays = tuple(brain.overlays.values())
+        control = ipv.animation_control(overlays,
+                                        len(time),
+                                        add=False,
+                                        interval=500)
+        slider = control.children[1]
+        slider.readout = False
+        slider.value = time_idx
+        label = widgets.Label(self._get_label(time[time_idx]))
+
+        # hadler for changing of selected time moment
+        def slider_handler(change):
+            k = brain.data['k']
+            b = brain.data['b']
+            lut = brain.data['lut']
+            time_idx_new = int(change.new)
+
+            for v in brain.views:
+                for h in brain.hemis:
+                    act_data = brain.data[h + '_array'][:, time_idx_new]
+                    smooth_mat = brain.data[h + '_smooth_mat']
+                    act_data = smooth_mat.dot(act_data)
+                    act_data = k * act_data + b
+                    act_data = np.clip(act_data, 0, 1)
+                    act_color_new = lut(act_data)
+                    brain.overlays[h + '_' + v].color = act_color_new
```

Comment:
```
        def slider_handler(change):
	            time_idx_new = int(change.new)
	
	            for v in brain.views:
	                for h in brain.hemis:
	                    smooth_mat = brain.data[h + '_smooth_mat']
	                    act_data = brain.data[h + '_array'][:, time_idx_new]
	                    act_data = smooth_mat.dot(act_data)
	                    act_data = brain.data['k'] * act_data + brain.data['b']
	                    act_data = np.clip(act_data, 0, 1)
	                    brain.overlays[h + '_' + v].color = brain.data['lut'](act_data)
```

It would actually be *more* readable like this I think.